docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,135 @@
+	- then the system type is also SYS-5: `G3_1_1d_expected_system_type = SYS_5`
+- all other cases: `else:`
+	- the system type is SYS-7: `G3_1_1d_expected_system_type = SYS_7`
+- loop through the zones_and_systems to see if any of the zones meets G3.1.1.d: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1d to determine if this zone meets this requirement: `if does_zone_meet_G3_1_1d(B_RMI,zone,zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] == YES):`
+		- change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1d"` 
+		- set the system to SYS_4: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = G3_1_1d_expected_system_type`
+
+- G3.1.1e "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+- loop through the zones_and_systems to see if any of the zones meets this requirement: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1e to determine whether to change the system type (we are using != NO instead of YES, because the function can return MAYBE.  Here we will assign any "maybe's to the new system type.  The rule itself will deal with the uncertainty of vestibules using the function is_zone_a_vestibule): `if does_zone_meet_G3_1_1e(P-RMI,B-RMI,zone) != NO:`
+		- - change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1e"`
+		- choose system 9 or 10 based on climate zone: `if is_CZ_0_to_3a():`
+			- set the system to SYS_10: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_10`
+		- else (the climate zone is_CZ_3b_3c_and_4_to_8): `else:`
+			- set the system to SYS_9: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_9`
+
+
+- G3.1.1f "If the baseline HVAC system type is 9 or 10, use additional system types for all HVAC zones that are mechanically cooled in the proposed design."
+- loop through the zones and systems, check only the zones that are SYS-9 or 10: `for zone in zones_and_systems:`
+	- check if it is SYS-9 or SYS-10: `if zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] in [SYS_9, SYS_10]:`
+		- use the function does_zone_meet_G3_1_1f to see if the zone meets the requirements of G3_1_1f: `if does_zone_meet_G3_1_1f(P-RMI,B-RMI,zone.id) == YES:`
+			- change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1f"`
+			- the zone meets the G3_1_1f requirements, choose system 3 or 4 based on climate zone: `if is_CZ_0_to_3a():`
+				- set the system to SYS_4: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_4
+			- else (the climate zone is_CZ_3b_3c_and_4_to_8): `else:`
+				- set the system to SYS_3: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_3`
```

Comment:
I think when exception f applies, we need to determine the baseline HVAC system for all such zones combined. So depending on occupancy type and size, it may be a multizone system. For example, if 1M SF warehouse has 100K SF taken by offices (but is not a mixed use building), the baseline system type for the office areas would be determined based on Table G3.1.1. This is clarified in 90.1 2022 G3.2.1.2 d: "d. If the baseline HVAC system type is 9 or 10, use additional system types for all HVAC zones that are mechanically cooled in the proposed design. The baseline HVAC system types for such zones shall be determined based on the building area type determined in accordance with Section G3.2.1.1(a) and the requirements of Section G3.2.1.1(c). "

Reply:
discussed on call - will select system type based on zone HVAC building area type (use `list_building_area_types_and_zones`), building total number of floors, and building total area

Reply:
updated